solidity/contracts/OwnableMulticall.sol

```diff
@@ -57,4 +61,19 @@ contract OwnableMulticall is OwnableUpgradeable {
             }
         }
     }
+
+    function resolveResults(address to, Result[] memory results) internal {
+        for (uint256 i = 0; i < results.length; i += 1) {
+            if (results[i].success) {
+                (bool success, bytes memory returnData) = to.call(
+                    results[i].data
+                );
+                if (!success) {
```

Comment:
I think it makes sense but imo call this out in both PR description and maybe code comments somewhere, both on `resolveResults` as well as `dispatch` on IQSRouter